components/google-cloud/google_cloud_pipeline_components/experimental/remote/gcp_launcher/custom_job_remote_runner.py

```diff
@@ -63,92 +63,79 @@ def create_custom_job(
   """
     client_options = {"api_endpoint": location + '-aiplatform.googleapis.com'}
     client_info = gapic_v1.client_info.ClientInfo(
-        user_agent="google-cloud-pipeline-components",
-    )
```

Comment:
not blocking - i saw the format back and forth multiple times. do we have a standard code formatter now after kfp sdk move its out?

Reply:
good point, I'm still using the code format from kfp namely via 
```
yapf  -i --style ~/pipelines/sdk/.style.yapf  ...<file_to_be_formatted>...
```
I think once we switch to Piper development we can use google lint as the standard format and formatter.